pkgs/by-name/pf/pflotran/package.nix

```diff
@@ -14,6 +14,25 @@
   python3,
 }:
 
+let
+  /*
+    Upstream petsc has lots of fortran api change since 3.22.0
+    We will keep using older version until pflotran supports the latest petsc.
+    Pflotran also requires Parmetis support in Petsc to have actual parmetis support.
+  */
+  petsc_ =
```

Comment:
```suggestion
  petsc' =
```

Reply:
Execuse me, but may I ask you not to arbitrarily override my changes in a package that I maintain, after I've outlined my reason to avoid adding a prime in a previous comment and chose the compromise solution? I think there is no clear guideline on how a variable _has_ to be named, no?